app/scripts/lib/experiments/grouping-rules/signup-password-confirm.js

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define((require, exports, module) => {
+  'use strict';
+
+  const BaseGroupingRule = require('./base');
+
+  const GROUPS = ['control', 'treatment'];
+
+  module.exports = class SignupPasswordConfirmGroupingRule extends BaseGroupingRule {
+    constructor () {
+      super();
+      this.name = 'signupPasswordConfirm';
+    }
+
+    choose (subject = {}) {
+      if (! subject.uniqueUserId && subject.forceExperiment !== this.name) {
+        return false;
+      }
+
+      if (subject.forceExperiment === this.name) {
```

Comment:
It seemed wasteful to go through the `q3FormChanges` machinery just to come right back here and force the experiment, when we already know that if the this boolean is true, the experiment is enabled.